README.md

```diff
@@ -0,0 +1,11 @@
+# Research project on amplitude models for N-star and other resonances in the GlueX experiment
```

Comment:
baryon resonances (?)

Reply:
I think not only baryon resonances, in the later stage, meson resonances are also considered, e.g. K* resonances.